lib/nimble_template/version.ex

```diff
@@ -15,6 +16,31 @@ defmodule NimbleTemplate.Version do
     end
   end
 
+  def upgrade_elixir_erlang_node_and_alpine(versions) do
```

Comment:
This function name is very descriptive but quite long. How about having a shorter name but moving the descriptive part to the argument:

```suggestion
  def upgrade_stack(elixir_erlang_node_alpine_versions) do
```

Reply:
Renamed the method in https://github.com/nimblehq/elixir-templates/pull/228/commits/0065a70cc1a28b8c3e0a440c6cb12cc4c284586e